pootle/apps/pootle_language/views.py

```diff
@@ -108,3 +112,26 @@ def language_admin(request, language):
     }
     return admin_permissions(request, language.directory,
                              'languages/admin/permissions.html', ctx)
+
+
+@get_path_obj
+@permission_required('administrate')
+def language_characters_admin(request, language):
+    if request.method == 'POST':
+        form = LanguageSpecialCharsForm(request.POST, instance=language)
```

Comment:
This seems it doesn't work.

Tried to use CBV here. Will have a further look at it.